selenium/che-selenium-core/bin/webdriver.sh

```diff
@@ -120,9 +117,10 @@ checkParameters() {
         if [[ "$var" =~ --web-driver-version=.* ]]; then :
         elif [[ "$var" =~ --web-driver-port=[0-9]+$ ]]; then :
         elif [[ "$var" == "--http" ]]; then :
-        elif [[ "$var" == "--https" ]]; then  :
+        elif [[ "$var" == "--https" ]]; then :
         elif [[ "$var" == "--che" ]]; then :
         elif [[ "$var" =~ --host=.* ]]; then :
+        elif [[ "$var" =~ --port=.* ]]; then :
```

Comment:
New parameter **--port** might be described in the next places as well:
- in the [README.md](https://github.com/eclipse/che/blob/master/selenium/che-selenium-test/README.md)
- in the [setup-selenium.md](https://github.com/eclipse/che-docs/blob/master/src/main/_docs/setup/setup-selenium.md)

Reply:
It is the `spi` branch, updating `setup-selenium.md` is not required.

Reply:
Thanks!
I've updated readme file. I won't update setup-selenium.md since it is in another repository where there is no spi branch.